data/AT.austria/AT11.burgenland/pipeline-specs.yaml

```diff
@@ -0,0 +1,3 @@
+AT11.burgenland:
```

Comment:
Why are pipeline-specs in git anyway? Aren't they supposed to be generated automatically?